templates/docker-compose.template

```diff
@@ -48,15 +45,13 @@ services:
       - {{I2C_DEVICE}}:{{I2C_DEVICE}}
     restart: on-failure
     environment:
```

Comment:
We need to add in:

    volumes:
      - pktfwdr:/var/pktfwd

Reply:
Right. something strange has happened while staging or rebasing.

Reply:
done